content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -407,22 +396,32 @@ Then create the check using sensuctl and the `disk_usage-check.json` file includ
 sensuctl create --file disk_usage-check.json
 {{< /highlight >}}
 
-We should see it working in the [dashboard entity view](http://localhost:3002/#/entities) and via sensuctl:
+_**NOTE**: You don't need to make any changes to disk_usage-check.json before running `sensuctl create --file disk_usage-check.json`._
+
+You should see the check working in the [dashboard entity view](http://localhost:3002/#/entities) and via sensuctl:
```

Comment:
should we be using https for these dashboard links?

Reply:
While `https` is typically preferred for anything that traverses a public network (the Internet), we don't need it for this lesson. All this traffic is via localhost (the user's computer) so any worry about non-encrypted traffic is low (if their computer is compromised, a Slack URL is the least of their worries). It would also require some non-default configuration changes on the Sensu Go end as it defaults to `http`, along with requiring a certificate or the user would get a warning about not being able to trust the site in their browser. Good question though!